.github/workflows/dapr-test.yml

```diff
@@ -120,17 +120,17 @@ jobs:
       - name: Deploy the test cluster
         if: env.TEST_PREFIX != ''
         run: |
-          # Select two random Azure regions
+          # Select one random Azure region
           REGIONS=(${{ env.AZURE_REGIONS }})
           REGIONS_SIZE=${#REGIONS[@]}
           REGIONS_IDX=$(($RANDOM % $REGIONS_SIZE))
           REGION1=${REGIONS[$REGIONS_IDX]}
           echo "REGION1=${REGION1}" >> $GITHUB_ENV
-          REGIONS_IDX=$(($RANDOM % $REGIONS_SIZE))
           REGION2=${REGIONS[$REGIONS_IDX]}
           echo "REGION2=${REGION2}" >> $GITHUB_ENV
-          echo "Deploying to Azure regions: Linux=${REGION1} Windows=${REGION2}"
-
+          REGION3=${{ env.AZURE_ARM_REGIONS }}
+          echo "REGION3=${REGION3}" >> $GITHUB_ENV
```

Comment:
Please also add below in the table in the message that there's this region too (see lines 173 and 188)